docs_user/modules/con_tlse-description.adoc

```diff
@@ -0,0 +1,18 @@
+[id="con_tlse-description_{context}"]
+
+= TLS Everywhere
+
+Note: The assumption is, that the new deployment will adopt the settings from the
+old deployment, so in case TLS Everywhere is disabled, it won't be enabled on
+the new deployment.
+
+If the Director deployment was deployed with TLS Everywhere, FreeIPA (IdM) is used
+to issue certificates for the OpenStack services. Certmonger, a client process which
+is installed on all hosts, interacts with FreeIPA (IdM) to request, install, track
+and renew these certificates.
+
+The new Operator based deployment uses the cert-manager operator to issue, track
+and renew the certificates.
```

Comment:
This raises a question in my mind: how does cert renewal work on data plane, as any Operator doesn't act on the data plane nodes directly? (Again, not a blocker.)

Reply:
There is an ansible playbook that runs when the secret changes, which deploys the certificate from the control plane into the data plane.  That's covered under the dataplane-operator TLS-E implementation.

Reply:
> There is an ansible playbook that runs when the secret changes

nope, user must run a deployment